index.html

```diff
@@ -0,0 +1,168 @@
+						<li class="menu__item"><a href="#" class="item__references">Features</a></li>
+					</ul>
+				</div>
+			</div>
+		</nav>
+		<header>
+			<div class="wrapper">
+				<article class="header__content">
+					<h1 class="header__title">Beautiful Free Nova template</h1>
+					<p class="header__text">A top notch premium quality template for your nex web project</p>
+					<button class="header__button--call-to-action">Download FREE!</button>
+				</article>
+				<div class="img-border">
+					<img src="./images/screen.png" alt="Image screen showcase" />
+				</div>
+			</div>
+		</header>
+		<main>
+			<section class="features">
+				<div class="wrapper">
+					<h2 class="features__title">Features</h2>
+					<div class="features__box">
+						<article class="features__element">
+							<div class="element__img element__img--responsive"></div>
+							<h3 class="element__title">Fully responsive</h3>
+							<p class="element__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Perferendis, voluptates.</p>
+						</article>
+						<article class="features__element">
+							<div class="element__img element__img--tools"></div>
+							<h3 class="element__title">Customizable</h3>
+							<p class="element__text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate, consequuntur.</p>
+						</article>
+						<article class="features__element">
+							<div class="element__img element__img--order"></div>
+							<h3 class="element__title">Slick and beautiful design</h3>
+							<p class="element__text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe, explicabo!</p>
+						</article>
+					</div>
+				</div>
+			</section>
+			<section class="companies-section none">
+				<figure class="companies">
+					<img src="./images/logos.png" alt="Companies logos" />
+				</figure>
+			</section>
+			<section class="imac-section">
+				<div class="wrapper">
+					<figure class="imac-section__img">
+						<img src="./images/imac.png" alt="Example picture of the imac" />
+					</figure>
+					<article class="text-section">
+						<h3 class="text-section__title">Simple and beautiful</h3>
+						<p class="text-section__text">
+							Lorem ipsum dolor sit amet, consectetur adipisicing elit. Beatae, modi necessitatibus. Officiis nam ipsum, incidunt earum assumenda maxime quasi
+							natus doloribus est aperiam sit culpa odit vero, neque voluptate repellendus.
+						</p>
+					</article>
+				</div>
+			</section>
+			<section class="pricing-section">
+				<div class="wrapper">
+					<div class="pricing-section__text">
+						<h3 class="pricing-section__text__title">choose your pricing plan</h3>
+						<p class="pricing-section__text__info">Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+					</div>
+					<div class="pricing-section__box">
+						<article class="box__pricing-item">
+							<p class="pricing-item__pricing-plan">basic</p>
+							<p class="pricing-item__pricing-value">$15</p>
```

Comment:
Raczej oceniałbym to jako istotne elementy więc zrobiłbym nagłówki np. `h4` i `h5`

Reply:
Okej, w takim razie, gdzieś kiedyś przeczytałem taką regułę, żeby nie skakać po nagłówkach + korzystać z trzech rozmiarów. Czyli używając h1 teoretycznie maksymalnym tagiem H jest tag h3. Jest to błędna reguła?